codecparsers/vc1Parser.cpp

```diff
@@ -232,87 +260,93 @@ namespace VC1 {
     {
         uint32_t i;
         BitReader bitReader(data, size);
```

Comment:
but all other function in the file use br pointer, i think it is not necessary to write another version of macro that use br instead of br pointer